toml/src/main/kotlin/org/rust/toml/crates/local/TestCratesLocalIndexServiceImpl.kt

```diff
@@ -10,8 +10,21 @@ import org.jetbrains.annotations.TestOnly
 class TestCratesLocalIndexServiceImpl : CratesLocalIndexService {
     var testCrates: Map<String, CargoRegistryCrate> = emptyMap()
 
+    private val _realInstance: CratesLocalIndexService by lazy {
+        CratesLocalIndexServiceImpl().apply {
+            updateIfNeeded()
+        }
+    }
+
```

Comment:
Let's move this code to `CratesLocalIndexServiceTest`. Currently, we use single static manually created object of service for testing. It's ok to speed up test execution but we need this instance only in `CratesLocalIndexServiceTest` and don't need it in `TestCratesLocalIndexServiceImpl`